### DISCUSSION TITLE
好像没有debug模式和配置热重载

### DISCUSSION THREAD

**Discussion by DongGua-Jun** [Arondight/Adachi-BOT#481] [General]
就是只响应主人命令

以及

发送指令私聊bot就可以让bot重新载入刚修改的配置

**Comment by Arondight** [comment]
debug 模式确实没有，我一般是单独申请一个QQ号来测试的，重载的话我本来想做成检测到ctime改变就重新读，不过感觉也没啥意义，就干脆让用户改完设置重启了

**Reply by DongGua-Jun** [reply]
这样啊，   另外就是 点歌的QQ音乐源好像坏掉了。  只提示歌曲查询出错

**Reply by mark9804** [reply]
> 点歌的QQ音乐源好像坏掉了

#434

**Reply by DongGua-Jun** [reply]
。。。  神特么丘丘音乐，我还以为丘丘人相关呢

**Reply by DongGua-Jun** [reply]
> 

查询 我的 XX时，角色的武器图片显示为error。是我的问题吗？
![JXN F847`B6EMF@5HR~ ~}3](https://user-images.githubusercontent.com/75875483/145680569-caea8030-06c9-431d-b323-f195b57f74b3.gif)
![DCUB96}46)IYAA(O9P)LCGR](https://user-images.githubusercontent.com/75875483/145680576-0b385c22-0c1f-49cf-94de-0991d3af7ff2.gif)

**Reply by mark9804** [reply]
> 是我的问题吗？

是的，武器图片是从米游社请求的，请检查一下你的设备连接米游社相关服务是否有困难

**Reply by DongGua-Jun** [reply]
我以为是我挂了VPN的原因，但是关了VPN再试，以及重启服务器再试也是不显示武器。

大佬怎么检查设备连接米游社相关服务是否有困难
![3R9W_TT5TZ(M5T$N4PT6B`U](https://user-images.githubusercontent.com/75875483/145681290-00137743-76f0-408b-adab-36aa43ac897c.jpg)

**Reply by DongGua-Jun** [reply]
> 
![image](https://user-images.githubusercontent.com/75875483/145681557-485c4c43-671c-421e-bbef-9dbe7f777d15.png)
ping好了，好像没啥问题

**Reply by DongGua-Jun** [reply]
> > 是我的问题吗？
> 
> 是的，武器图片是从米游社请求的，请检查一下你的设备连接米游社相关服务是否有困难

![image](https://user-images.githubusercontent.com/75875483/145681634-a5aebde8-1990-493a-8caa-d678bfb059c2.png)
![5DYDVKASVZ{PEE C~70K_9H](https://user-images.githubusercontent.com/75875483/145681661-18dcec6d-6bc4-4480-8c35-2d12799eccb3.jpg)

寄续当咸鱼了

**Reply by mark9804** [reply]
Windows的话，我不太清楚是不是Windows的问题，你可以试试看
```
wget https://upload-bbs.mihoyo.com/game_record/genshin/equip/UI_EquipIcon_Pole_Homa.png
```
```
ping /n 20 upload-bbs.mihoyo.com
```
第二个ping我不知道是不是这样的，已经拿Windows只做游戏机很久了，现在也不想特意开机就为跑一个命令

**Comment by DongGua-Jun** [comment]
![image](https://user-images.githubusercontent.com/75875483/145681923-02bc9770-5aa4-4237-bdef-056ae04dff0f.png)
没丢包   先前是网络波动了吧
![W3GS$EB_WTPE1$L@}@JYH2W](https://user-images.githubusercontent.com/75875483/145681946-d2a9879c-390f-4ac5-bd36-3727ca5e48e7.jpg)
